cohort-evaluator-spark/src/test/java/com/ibm/cohort/cql/spark/SparkCqlEvaluatorTest.java

```diff
@@ -818,7 +818,126 @@ public void testValidateSameDefinesWithOutputMapping() throws Exception {
                 "parquet"
         );
     }
-    
+
+    @Test
+    public void testAbstractContextChoiceModel() throws Exception {
+        File inputDir = new File("src/test/resources/abstract-context/");
+        File outputDir = new File("target/output/abstract-context/");
+
+        File output = new File(outputDir, "AlphaNumeric");
+
+        String [] args = {
+            "-d", "src/test/resources/abstract-context/metadata/context-definition-choice.json",
+            "-j", "src/test/resources/abstract-context/metadata/cql-jobs-choice.json",
+            "-m", "src/test/resources/abstract-context/modelinfo/abstract-modelinfo-1.0.0.xml",
+            "-c", "src/test/resources/abstract-context/cql",
+            "--input-format", "parquet",
+            "-i", "AlphaNumeric=" + new File(inputDir, "data/One.parquet").toURI(),
```

Comment:
Is it possible to have multiple valid `AlphaNumeric` types at once when the engine runs? Could you run the given CQL against both data of types `One` and `A` in one go?

Reply:
Right, you could.
If you ran over both `One` and `A` with the example `CQL`, however, you would get a `RuntimeException`, as `unit` isn't part of the `A` schema.

Reply:
Could you do a single run of the engine that processes both `One` and `Two`?

Reply:
Does that syntax work with the `-i` option?

Reply:
Good question.

Yes, that is a valid Hadoop [FileSystem::globStatus](https://hadoop.apache.org/docs/current/api/org/apache/hadoop/fs/FileSystem.html#globStatus-org.apache.hadoop.fs.Path-) pattern.

> {ab,cd}
Matches a string from the string set {ab, cd}

Reply:
Would you mind setting up the test to read from multiple files? I'm having trouble trying to get this to work on the branch I have checked out, but I might just be doing something silly.

Reply:
addressed in 4dfffa9